hack/clean.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# **clean.sh**
+
+cd $GOPATH/src/github.com/kubeedge/kubeedge
+
+removefolder() {
+if [ -d bin ]; then
+rm -rf bin
+fi
+
+if [ -d .idea ]; then
```

Comment:
Expectation is we take care of only kubeedge related files, folders and binaries. This repo should not take care of IDE related issues. IDEA is developers own development env. Hence not related here. Remove these lines

Reply:
ok